### PR TITLE
Sky migration to cosmwasm v1

### PR DESCRIPTION
**Pull request by SissonJ**
Migrating sky to cw1

### CODE REVIEW

File: contracts/sky/Cargo.toml [hunk 1]
```diff
@@ -24,7 +24,8 @@ default = []
 backtraces = ["shade-protocol/backtraces"]
 
 [dependencies]
-shade-protocol = { version = "0.1.0", path = "../../packages/shade_protocol", features = ["sky", "sky-impl", "adapter", "math"] }
+shade-protocol = { version = "0.1.0", path = "../../packages/shade_protocol", features = ["sky", "sky-impl", "adapter", "math", "shade_admin"] }
```
Comment (FloppyDisck) [inline contracts/sky/Cargo.toml:27]: you can make `sky-impl` import all the necessary features such as `sky` and `shade_admin` so you only have to import the impl feature and done

File: packages/shade_protocol/Cargo.toml [hunk 2]
```diff
@@ -88,7 +88,7 @@ backtraces = ["cosmwasm-std/backtraces"]
 debug-print = [] # TODO: remove this from all cargo configs
 
 [dependencies]
-cosmwasm-std = { git = "https://github.com/scrtlabs/cosmwasm", branch = "secret" }
+cosmwasm-std = { git = "https://github.com/scrtlabs/cosmwasm", branch = "secret", features = ["staking"] }
```
Comment (FloppyDisck) [inline packages/shade_protocol/Cargo.toml:91]: you should leave the staking feature out and import it whereever you need it

File: contracts/sky/Cargo.toml (hunk 1 above)
Comment (SissonJ) [reply contracts/sky/Cargo.toml:27]: Sky, sky-impl, and shade-admin should remain separate since eventually will refactor sky-impl into a dex helper

Comment (FloppyDisck) [reply contracts/sky/Cargo.toml:27]: yes but sky-impl relies on the sky feature

sky acts like the interface and the impl is functionality, if you cant import the impl without importing sky then they should be automatically imported

**Review by FloppyDisck — APPROVED** [review]

**Pull request merged**